src/StateL1Null.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @file
+ * Null implementation for L1 state holder.
+ */
+
+namespace LCache;
+
+/**
+ * Fake/Stub state manager class.
+ *
+ * @author ndobromirov
+ */
+class StateL1Null extends StateL1Static implements StateL1Interface
```

Comment:
I felt something was off when I was implementing this. The only state implementation that caused issues was the NULL one. I think that state should be ok to be the static one. Will change it so.